octokit-plugins/logger.js

```diff
@@ -25,7 +25,7 @@ function log(state, level, ...args) {
 
   if (obj.msg && (level !== "debug" || state.options.debug)) {
     const { msg: message, ...data } = obj;
-    state.onLogMessage(level, message, data);
+    state.onLogMessage(level, message, { ...state.context, ...data });
```

Comment:
I wonder if this might be too noisy in the stdout logs? We already log out the current repository before we run the script on it. It might be redundant information. That's why I left it out here

Reply:
I'm okay either way, but slightly prefer not to add `state.context` here.

But we could add a comment above

https://github.com/octoherd/octokit/blob/0784191b31c0fb3ef1fe46123556fe4a1b616e17/octokit-plugins/logger.js#L48

to clarify that the context is only added to the log file, not the stdout

Reply:
Sounds good to me